content/en/security/application_security/getting_started/serverless.md

```diff
@@ -0,0 +1,147 @@
+---
+title: AWS Lambda Getting Started with ASM
+kind: documentation
+is_beta: true
+code_lang: serverless
+type: multi-code-lang
+code_lang_weight: 90
+further_reading:
+    - link: "/security/application_security/how-appsec-works/"
+      tag: "Documentation"
+      text: "How Application Security Works"
+    - link: "/security/default_rules/#cat-application-security"
+      tag: "Documentation"
+      text: "OOTB Application Security Management Rules"
+    - link: "/security/application_security/troubleshooting"
+      tag: "Documentation"
+      text: "Troubleshooting Application Security Management"
+    - link: "/security/application_security/threats/"
+      tag: "Documentation"
+      text: "Application Threat Monitoring and Protection"
+    - link: "/security/application_security/risk_management/"
+      tag: "Documentation"
+      text: "Application Risk Management"
+---
+
+{{< callout url="#" header="ASM support for AWS Lambda is in beta" btn_hidden="true" >}}
+  Threat detection is done by using the Lambda extension.
```

Comment:
Above this, insert a note to link to the Compatibility section so customers can double check they're using the intended capabilities

Text should be "Please take a look at the [Compatibility page](https://docs-staging.datadoghq.com/kari/docs-4776-asm-serverless/security/application_security/threats/setup_and_configure/#compatibility) for what ASM supports for AWS Lambda"